### ISSUE TITLE
Is there way to skip over doc links in comments?

### ISSUE THREAD

**Issue by stusmall** [drahnr/cargo-spellcheck#294]
In my project I'm making heavy use of links to other types in doc strings.  Because of the capitalization of rust type names, it can cause a lot of false positives.  For example:

```
error: spellcheck(Hunspell)
   --> /home/stusmall/Workspace/bowbend/bowbend_core/src/service_detection/framework/rule_results.rs:12
    |
 12 |  Returns the [RuleId] of the Rule this result belongs to.  This isn't
    |               ^^^^^^
    | - Rule Id, Ruled, or Rule-id
    |
    |   Possible spelling mistake found.
```
Does cargo spellcheck have a way to skip over these links?  Or is this better handled by a rule in the dictionary file?  If so do you have a rule handy to ignore anything inside brackets?

Once again, huge thank you for this project. It's helped keep my documentation much cleaner.

**Comment by stusmall** [comment]
Right after posting this I saw that this is mentioned in the [FAQ](https://github.com/drahnr/cargo-spellcheck/blob/master/docs/remedy.md#types-in-doc-comments)

**Comment by drahnr** [comment]
It's always good to know people read the FAQ _and_ it actually contains useful info :+1:
